toolkit/recordreplay/ProcessRecordReplay.cpp

```diff
@@ -308,7 +308,7 @@ MOZ_EXPORT void RecordReplayInterface_Initialize(int* aArgc, char*** aArgv) {
   MOZ_RELEASE_ASSERT(dispatchAddress.isSome());
 
   Maybe<std::string> apiKey;
-  const char* val = getenv("RECORD_REPLAY_API_KEY");
+  const char* val = getenv("RECORD_REPLAY_AUTH");
```

Comment:
I don't think this is right, this will cause the browser to no longer recognize RECORD_REPLAY_API_KEY.  I think we need to check for both environment variables here.

Reply:
RECORD_REPLAY_API_KEY will also be copied to RECORD_REPLAY_AUTH during initialization, just like the user's token is copied to it if there is no API key. Both happen in the `setAccessToken()` function now.

Reply:
Cool, thanks!  It would be nice to have some comments on this logic to explain what's going on (echoing my other comment on the patch).